MontyHallv2/DoorCreation/Door.cs

```diff
@@ -0,0 +1,42 @@
+namespace MontyHallv2.DoorCreation;
+
+public class Door
+{
+    private bool _opened;
+    private bool _hasCar;
+    private bool _playerPicked;
+
+    public void OpeningDoor()
```

Comment:
```suggestion
    public void Open()
```

reads nicer when called e.g. `_door.Open()` or `_door.OpenDoor()`